src/Microsoft.Fx.Portability/ApiPortService.cs

```diff
@@ -219,5 +168,35 @@ public void Dispose()
                 return formatInformation;
             }
         }
+
+        private static HttpMessageHandler BuildMessageHandler(string endpoint, IProxyProvider proxyProvider)
+        {
+            if (string.IsNullOrWhiteSpace(endpoint))
+            {
+                throw new ArgumentOutOfRangeException(nameof(endpoint), endpoint, LocalizedStrings.MustBeValidEndpoint);
+            }
+
+            // Create the URI directly from a string (rather than using a hard-coded scheme or port) because 
+            // even though production use of ApiPort should always use HTTPS, developers using a non-production
+            // portability service URL (via the -e command line parameter) may need to specify a different 
+            // scheme or port.
+            var uri = new Uri(endpoint);
+
+            var clientHandler = new HttpClientHandler
+            {
+#if !FEATURE_SERVICE_POINT_MANAGER
```

Comment:
I just noticed this. Do you remember why FEATURE_SERVICE_POINT_MANAGER was only defined for .NET 4.6 but not .NET Standard?

Reply:
The way .NET Framework and .NET Standard handle SSL is different. @mjrousos could comment better on that